db/seeds/application_types.yml

```diff
@@ -19,6 +19,7 @@
   :supported_authentication_types:
     amazon:
     - arn
+    - superkey
```

Comment:
So as I proposed in DMs, we don’t need to store which applications support the superkey, as we should support the feature for whole types. (Now, as we are talking just about AWS type, both its applications shares the same config credentials, so there will be not so much work more to do to support both of them.)

Reply:
Yep, sorry for taking so long on this, it also doesn't make sense to have this here because the superkey authentication is based on the `Source` record, not the application. Whoops!